src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -55,21 +55,18 @@
       :mention
       (conj
        units
-       [rn/view
-        {:style style/mention-tag-wrapper}
-        [rn/touchable-opacity
-         {:active-opacity 1
-          :on-press       #(rf/dispatch [:chat.ui/show-profile literal])
-          :style          style/mention-tag}
-         [quo/text
-          {:weight :medium
-           :style  style/mention-tag-text
-           :size   :paragraph-1}
-          (rf/sub [:messages/resolve-mention literal])]]])
+       [rn/touchable-opacity
+        {:active-opacity 1
```

Comment:
:active-opacity is redundant

Reply:
It was set this way in the past, and I believe the reason is that we don't require an opacity effect when a user clicks on a mention.

Reply:
@OmarBasem can you remember any reason for set active-opacity or i can remove it? i believe it was part of this PR https://github.com/status-im/status-mobile/pull/15799

Reply:
`:active-opacity 1` is not redundant as the default `active-opacity` is 0.5. However, we are moving away from `touchable-opacity`, so you can replace it with `rn/pressable`

Reply:
resolved